pulp_container/app/models.py

```diff
@@ -371,6 +373,9 @@ class ContainerDistribution(
         related_name="container_distributions",
         null=True,
     )
+    private = models.BooleanField(
+        default=False, help_text=_("Prevent repository to be pulled by anonymous users.")
```

Comment:
we should phrase this somehow differently, if repo is private no only anonymous user cannot pull, but as well any authed user unless explicitly added by the owner.

Reply:
```suggestion
        default=False, help_text=_("Restrict pull access to explicitly authorized users. Defaults to unrestricted pull access.")
```
Like that?

Reply:
:fire: